openml-api/src/test/java/com/feedzai/openml/data/schema/DatasetSchemaTest.java

```diff
@@ -57,9 +58,15 @@ public void testInvalidTargetIndex() {
      */
     @Test
     public final void testNoTargetIndex() {
-        assertThat(new DatasetSchema(ImmutableList.of(FIELD_SCHEMA)).getTargetIndex())
+        final SoftAssertions assertions = new SoftAssertions();
```

Comment:
i see soft assertions more useful when using loops, otherwise every single UT with +1 assertion would use this, but ok :)

Reply:
Agree with the first statement, but SoftAssertions also give you another advantage: lazy return (of assertion results). In this case both assertions are independent, and as such I want to evaluate the second even if the first one fails. However, you are close to a critical problem: I'm missing the `assertions.assertAll()` :P

Reply:
I agree with Rijo's point;  no big deal however.  In practice what just happened is a good reason to not use SoftAssertions unless really needed (i.e., it is easy to forget to assert :) )